packages/site-server/package.json

```diff
@@ -29,13 +30,20 @@
         "src/**/*.ts"
       ],
       "output": []
+    },
+    "serve:dev": {
```

Comment:
Do we want this? It's already in the top-level. I kind of lean towards moving this sort of top level commands entirely to the top level, rather than doing delegation. But currently the `node` command is duplicated, so I think we want one or the other (all in top-level, or actual delegation).

Reply:
I was going to have the command in the package be the canonical one, but before wireit service mode it's a little trickier to call. Once we have service mode I think we _do_ want them in the packages to be local, and top-level scripts that coordinate the package-level scripts.